Dockerfile

```diff
@@ -1,27 +1,26 @@
 # Dockerfile # Mapknitter
 # https://github.com/publiclab/mapknitter/
+# This image deploys Mapknitter!
 
 FROM ruby:2.4.4-stretch
-MAINTAINER Sebastian Silva "sebastian@fuentelibre.org"
-
-LABEL This image deploys Mapknitter!
 
 # Set correct environment variables.
-RUN mkdir -p /app
 ENV HOME /root
 
 # Install dependencies
 RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
-RUN apt-get update -qq && apt-get install -y bundler default-libmysqlclient-dev ruby-rmagick libfreeimage3 libfreeimage-dev ruby-dev gdal-bin python-gdal curl libcurl4-openssl-dev libssl-dev zip nodejs ##ALSO TRIED: ruby-pg
-RUN npm install -g bower
+RUN apt-get update -qq && apt-get install -y default-libmysqlclient-dev \
+                                             ruby-rmagick libfreeimage3 \
+                                             libfreeimage-dev ruby-dev \
+                                             gdal-bin python-gdal curl \
+                                             libcurl4-openssl-dev libssl-dev \
+                                             zip nodejs ##ALSO TRIED: ruby-pg
 
-# Install bundle of gems
-WORKDIR /tmp
-ADD Gemfile /tmp/Gemfile
-ADD Gemfile.lock /tmp/Gemfile.lock
-RUN bundle install
```

Comment:
Abstracted this to a script, since RUN commands creates a new layer thus increasinf docker image size.